rtv/subreddit.py

```diff
@@ -106,23 +106,19 @@ def open_submission(self):
         page = SubmissionPage(self.stdscr, self.reddit, url=data['permalink'])
         page.loop()
 
-        if data['url'] == 'selfpost':
```

Comment:
Don't forget this function also gets called when you press the right or "l" key, we want to add to the history in that case too